assets/js/payplug-admin-bancontact.js

```diff
@@ -61,6 +66,7 @@
 		},
```

Comment:
add: 
jQuery("#bancontact_live_mode_description_disabled").show();
to disableBancontact

Reply:
with this aproach you'll have:
 - Default behaviour show the comment
 - once the ajax call returns, you can show/hide according to the response
 - less code :)

Reply:
Done
After move this line to disableBancontact(),  I have just added a condition before it 
because if not it will appear in test mode also and we will have 2 warning messages like in the image :
https://ibb.co/J37ph0W